### PR TITLE
Update committers.md

### PR DESCRIPTION
**Pull request by manez**
**GitHub Issue**: #1040 

# What does this Pull Request do?

Very minor clean-up on the Committers.md page. Removing some ?? marks. 

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

File: docs/contributing/committers.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ Summary:
 
 If they accept, then do:
 
-1. Add to Committer team of GitHub Islandora organization ??
```
Comment (whikloj) [inline docs/contributing/committers.md:88]: With the split in Islandora 7 and 8, should this be 
> Add to the Islandora 8 Committer team of the Github Islandora organization.

File: docs/contributing/committers.md [hunk 2]
```diff
@@ -85,8 +85,8 @@ Summary:
...
-2. Add to Committer team of GitHub Islandora-Labs organization ??
```
Comment (whikloj) [inline docs/contributing/committers.md:89]: What does this team do? I don't think committers as a team have any greater access then a regular user, unless granted by... you maybe?

Comment (whikloj) [reply docs/contributing/committers.md:89]: I'm sorry, we probably have write permissions. I wasn't thinking clearly. Nevermind this one.

File: docs/contributing/committers.md (hunk 1 above)
Comment (manez) [reply docs/contributing/committers.md:88]: Right-o!

**Review by whikloj — APPROVED** [review]
👍

**Pull request merged**